### PR TITLE
Services Folder- User.js (App.js changes)

### PR DESCRIPTION
**Pull request by natesanchez**
- Description of changes made: 
1.) Constants file setup for production vs Local URL
2.) Functions from Admin.js moved to User.js in services folder

- Is the feature complete/bug resolved/etc..: Bug at first.. but got frontend to run locally.

- Any known bugs/strange behavior: Not that I'm aware of.

FeedBack: In VS Code my function imports in App.js from my User.js file appear greyed out when my binding looks like this (btw this doesn't have Python syntax anymore):

Screen Shot 2020-04-09 at 8 40 26 AM
Screen Shot 2020-04-09 at 8 40 39 AM

But look fine when I change up the binding, removing "this" after the equal sign. Just want to make sure this is correct syntax? I saw something on stack overflow regarding this and function imports, but just wanted to be sure.:

Screen Shot 2020-04-09 at 8 39 40 AM

Screen Shot 2020-04-09 at 8 39 54 AM

### CODE REVIEW

**Comment by saralaffin** [comment]
I hope y'all don't mind I added to this PR, based on a comment from @ZakkMan on another review (that I can't seem to find right now..)

**Comment by natesanchez** [comment]
@saralaffin 👍🏽

File: client/src/App.js [hunk 1]
```diff
@@ -21,11 +21,16 @@ import {
...
-  faEdit,
+  faEdit
 } from "@fortawesome/free-solid-svg-icons";
 import { fab } from "@fortawesome/free-brands-svg-icons";
 import { fas } from "@fortawesome/free-solid-svg-icons";
 import { far } from "@fortawesome/free-regular-svg-icons";
+import { BASE_URL } from "./services/constants";
+import { handleSignup } from "./services/User";
+import { handleLogin } from "./services/User";
+import { handleLogout } from "./services/User";
```
Comment (ZakkMan) [inline client/src/App.js:32]: @natesanchez FYI you can merge these:

```js
import { handleSignup, handleLogin, handleLogout } from "./services/User";
```

Comment (natesanchez) [reply client/src/App.js:32]: @ZakkMan sure thing, I think I know what it could be

**Pull request merged**